app/views/CorporationTaxEnquiriesView.scala.html

```diff
@@ -0,0 +1,63 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import views.html._
+
+@this(
+    main_template: MainTemplate,
+    appConfig: AppConfig)
+
+@()(implicit request: Request[_], messages: play.api.i18n.Messages)
+
+@main_template(title = messages("global.nav.title")) {
+
+@components.back_link()
+
+    <h1 class="header-margin-top">@messages("corporation.tax.enquiries.title")</h1>
+
+    <div class="govuk-inset-text">
+        <p>@messages("global.opening.times.title")</p>
+        <p>@messages("global.opening.times.8_30to4")</p>
+        <p>@messages("global.opening.times.advisers.p")</p>
+    </div>
+
+    @if(!appConfig.performanceTest){
+        <div id="HMRC_FIXED_1"></div>
+    }
+
+    <p>@messages("corporation.tax.enquiries.p1")</p>
+    <p>@messages("corporation.tax.enquiries.p2")</p>
+
+    <div>
+        <li>@messages("corporation.tax.enquiries.bullet.1")</li>
+        <li>@messages("corporation.tax.enquiries.bullet.2")</li>
+        <li>
+            <a id="gov-gateway-link" href=@appConfig.signInGovernmentGatewayUrl>@messages("global.text.link.online")</a>
+            @messages("corporation.tax.enquiries.bullet.3")
+        </li>
+    </div>
+
+    <div class="margin-top">
+        @messages("corporation.tax.enquiries.p3.1")
+        <a id="gov-gateway-link" href=@appConfig.askForCorporationUTRUrl>@messages("corporation.tax.enquiries.p3.2")</a>
+        @messages("corporation.tax.enquiries.p3.3")
+    </div>
+
+    <p class="margin-top">
+        <a id="return-link" href=@appConfig.corporationTaxEnquiriesReturnUrl>@messages("global.text.link.back.to.govuk.p1")</a>
+    </p>
```

Comment:
prototype text say "Return to contact HMRC page" however not sure if having the word 'page' is correct ?

Reply:
the prototype is incorrect all the other pages in the service say "Return to contact HMRC"